### PR TITLE
Added note on usage of "netplan" for Ubuntu network configuration

### PR DESCRIPTION
**Pull request by Hudratronium**
As since focal release the standard for network configuration is NetPlan,  i suggest a information in the guide until examples for the configuration are availeable.

### CODE REVIEW

File: source/installguide/hypervisor/kvm.rst [hunk 1]
```diff
@@ -598,6 +598,11 @@ we only have to configure the network.
 
    $ vi /etc/network/interfaces
 
+.. note::
+   Since Ubuntu 20.04 the standard for mangeing network connections is by
+   useing NetPlan YAML files. Please refer to the Ubuntu man pages for further
+   information and set up network connections figuratively.
```
Comment (GabrielBrascher) [inline source/installguide/hypervisor/kvm.rst:604]: 1. `mangeing` -> `managing`
2.  `useing`  -> `using`

File: source/installguide/hypervisor/kvm.rst [hunk 2]
```diff
@@ -768,6 +773,11 @@ Configure Ubuntu for Advanced Networks
 All the required packages were installed when you installed libvirt, so
 we only have to configure the network.
 
+.. note::
+   Since Ubuntu 20.04 the standard for mangeing network connections is by
+   useing NetPlan YAML files. Please refer to the Ubuntu man pages for further
+   information and set up network connections figuratively.
```
Comment (GabrielBrascher) [inline source/installguide/hypervisor/kvm.rst:779]: Same as above

**Review by GabrielBrascher — COMMENTED** [review]
Thanks for the PR @Hudratronium. I added a few adjustments.

**Review by DaanHoogland — COMMENTED** [review]
looks good, but it is duplicate code. can we solve this in another way?

**Comment by Hudratronium** [comment]
@DaanHoogland 
Moved the "Note" up to the introductional part of the whole networking chapter. As long as one is going through the guide step by step he should be reading and seeing it.

**Review by DaanHoogland — APPROVED** [review]

**Pull request merged**